kong/runloop/handler.lua

```diff
@@ -408,6 +413,23 @@ return {
         balancer.init()
       end)
 
+      do
+        local err
+
+        -- semaphore (mutex) for build_api_router
+        build_api_router_semaphore, err = semaphore.new()
+        if err then
+          log(ERR, "failed to create build_api_router_semaphore: ", err)
```

Comment:
This log is still using the `ngx.ERR` level instead of `CRIT` - mind updating it?

Reply:
Shoot, my bad!